src/styles/css/main.css

```
@@ -0,0 +1,345 @@
+        width: 100%;
+        padding: 4.2rem 6.2rem 3rem 9.5rem;
+        max-width: 1440px;
+    }
+    .navbar__menu {
+        position: static;
+        transform: none;
+        height: auto;
+        width: auto;
+        padding: 0;
+        order: 3;
+        transform: none;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

transform is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)